gauntlet/packages/gauntlet-solana/src/commands/middlewares.ts

```diff
@@ -20,11 +22,21 @@ export const withProvider: Middleware = (c: SolanaCommand, next: Next) => {
   return next()
 }
 
-export const withWallet: Middleware = (c: SolanaCommand, next: Next) => {
+export const withWallet: Middleware = async (c: SolanaCommand, next: Next) => {
+  if (c.flags.withLedger || boolean(process.env.WITH_LEDGER)) {
```

Comment:
if `--withLedger` flag is provided or `WITH_LEDGER` env var is set, tries to load Ledger